larky_server/protos/larky.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+option java_package = "com.verygood.security.messages.operations.starlark";
+
+message StarlarkRequest {
+  string script = 1;
+  string payload = 2;
+  string main_fn = 3;
+  map<string, string> ctx = 7;
+}
+
+
+message StarlarkResponse {
+  string payload = 1;
+  map<string, string> ctx = 2;
+}
+
+
+service ComputeService {
```

Comment:
Hi, it is great to see the starlark server coming together :)
Please, consider renaming this service something more specific, like LarkyRuntimeService. We have a lot of stuff prefixing with Compute already and I fear that it will mislead people and overload the term.

Reply:
@Agitolyev done!